db.php

```diff
@@ -0,0 +1,47 @@
+<?php
+    class dbconnect {
+        public $host;
+        public $user;
+        public $pass;
+        public $db;
+        public $conn;
+
+        public function __construct() {
+            $this->host = "localhost";
+            $this->user = "root";
+            $this->db = "bits";
+            $this->pass = "";
+        }
+        function connect(){
+            $this->conn = mysqli_connect($this->host,$this->user,$this->pass,$this->db);
```

Comment:
thiếu space VD: `mysqli_connect($this->host, $this->user, $this->pass, $this->db);`